sncosmo/models.py

```diff
@@ -1948,15 +1953,24 @@ def minwave(self):
     def maxwave(self):
         return self._maxwave
 
+    def minphase(self):
+        return self._minphase
+
+    def maxphase(self):
+        return self._maxphase
```

Comment:
Is it possible to have L1956-1960 contain the `np.nan` default values, rather than adding them to all previously built models? I think this might be
```python
try:
    return self._maxphase
except NameError:
    return np.nan
```

This might also help with  backwards compatibility. If this works, then we can remove the changes to `CCM89Dust`, `OD94Dust`, and L1570-1573.

Reply:
Sure I think that would work let me do that.